lib/util/index.coffee

```diff
@@ -84,9 +84,14 @@ exports.noApplicationForKeyResponse =
 	statusCode: 500
 	body: 'No application found to associate with the api key'
 
-exports.uniqueKeyViolated =
-	code: 'BalenaRequestError'
-	body: 'Unique key constraint violated'
+exports.isUniqueKeyViolationResponse = ({code, body}) ->
+	code == 'BalenaRequestError' &&
+	(
+		# api translated response
+		body == 'Unique key constraint violated' ||
+		# pine response (tested on pine 10)
+		/^".*" must be unique\.$/.test(body)
```

Comment:
This is good enough as it is (I mean the “.*” bit), but you could also provide as an argument the actual field names you expect constraints to fail and make it a lot more robust. Cloud API needs to translate constraint violations out of bounds, hence the need to cast such a wide net with the regular expression.

Reply:
Maybe we can re-iterate on this on the orgs sdk, since the v6 endpoint will probably no longer translate the pine error to `Unique key constraint violated`.

Reply:
I see. Yeah, it’s good as it is :)